server/opts.go

```diff
@@ -92,7 +92,7 @@ type GatewayOpts struct {
 	Advertise      string               `json:"advertise,omitempty"`
 	ConnectRetries int                  `json:"connect_retries,omitempty"`
 	Gateways       []*RemoteGatewayOpts `json:"gateways,omitempty"`
-	RejectUnknown  bool                 `json:"reject_unknown,omitempty"`
+	RejectUnknown  bool                 `json:"reject_unknown,omitempty"` // config got renamed to reject_unknown_cluster
```

Comment:
If you want, you can add this same comment to GatewayOptsVarz object in monitor.go.

Reply:
ok, wil do

Reply:
fixed